probe_ble_specification.rst

```diff
@@ -0,0 +1,290 @@
+
+The probe status mentioned in the above service is described here:
+
+.. _probe status:
+
+============================ ======== ===== ===========================================================================================
+Value                        Format   Bytes Description
+============================ ======== ===== ===========================================================================================
+Log Range                    uint32_t 8     Range of logs available on the probe. Two ``uint32_t`` sequence numbers (``min``, ``max``).
+Current Raw Temperature Data uint8_t  13    See `Raw Temperature Data`_.
+Mode/ID                      uint8_t  1     See `Mode and ID Data`_.
+Status                       uint8_t  1     See `Device Status`_.
+============================ ======== ===== ===========================================================================================
+
+UART Service
+------------
+
+The UART service is a custom BLE service that emulates a UART. The UUID for the
+UART service is ``6E400001-B5A3-F393-E0A9-E50E24DCCA9E``.
+
+The RX characteristic is used to receive data and the TX characteristic is used
+to transmit data via BLE notifications. The format of the data sent and
+received over this service is described in the `UART Messages`_ section.
+
+============== ======================================== ========================================================== ===========
+Characteristic UUID                                     Description                                                Properties
+============== ======================================== ========================================================== ===========
+RX             ``6E400002-B5A3-F393-E0A9-E50E24DCCA9E`` Peer device can send data to Probe on RX characteristic.   Write
+TX             ``6E400003-B5A3-F393-E0A9-E50E24DCCA9E`` Probe can send data to a peer device on TX characteristic. Read/Notify
+============== ======================================== ========================================================== ===========
+
+Device Firmware Update Service
+------------------------------
+
+The Device Firmware Update (DFU) Service is a custom service provided by Nordic
+service for updating the firmware on the Predictive Probe.
+
+Details TBD.
+
+UART Messages
+#############
+
+The section describes the protocol that will be sent and received over the
+Nordic UART Service.
+
+Command Header
+--------------
+
+Each message will begin with the same 5 byte header, followed by the message
+payload. The payload of each message type is described below.
+
+============== ======== ===== ===================================================================
+Value          Format   Bytes Description
+============== ======== ===== ===================================================================
+Sync Bytes     uint8_t  2     ``{ 0xCA, 0xFE }``
+Checksum       uint16_t 2     Checksum for message type, payload length + payload (sum of bytes).
```

Comment:
Should we indicate that this is unused?  So that it matches what is currently implemented?